subgraphs/goldfinch/src/mappings/senior_pool.ts

```diff
@@ -355,6 +355,9 @@ export function handleInvestmentMadeInSenior(
   market.totalBorrowBalanceUSD =
     market.totalBorrowBalanceUSD.plus(newBorrowUSD);
   market.cumulativeBorrowUSD = market.cumulativeBorrowUSD.plus(newBorrowUSD);
+  if (!market._interestTimestamp) {
```

Comment:
It would save all these if not statements if you set the interest timestamp at market creation

Reply:
@dmelotik  Right! But there is a certain migrated tranched pool that does not have the market creation event, so I am adding this statement to handle it.